File: decompiled/sources/com/streamax/client/ui/devlist/db/GroupDao.java
package com.streamax.client.ui.devlist.db;

import java.util.List;

public abstract class GroupDao {
    public abstract boolean add(String str, int i, String str2, int i2, int i3);

    public abstract boolean delete(int i);

    public abstract boolean delete(int i, int i2);

    public abstract boolean delete(String str);

    public abstract boolean delete(String str, int i, String str2, int i2);

    public abstract void deleteDb();

    public abstract int getChCount(String str, int i);

    public abstract List<GroupBean> getGroupDatas();

    public abstract List<GroupBean> getGroupDatasByName(String str);

    public abstract boolean isEmpty();

    public abstract boolean query(String str);

    public abstract boolean query(String str, int i);

    public abstract boolean query(String str, int i, String str2, int i2);

    public abstract boolean queryByGroupAndDevName(String str, String str2, int i);

    public abstract boolean queryDevName(String str);

    public abstract String queryDevNameByDevId();

    public abstract boolean update(String str, int i);

    public abstract boolean update(String str, int i, String str2, int i2, int i3);

    public abstract boolean update(String str, String str2);

    public abstract boolean updateDevNameById(int i, String str);
}

File: decompiled/sources/com/streamax/client/ui/devlist/db/GroupDao.kt
package com.streamax.client.ui.devlist.db

abstract class GroupDao {
    abstract fun add(str: String, i: Int, str2: String, i2: Int, i3: Int): Boolean

    abstract fun delete(i: Int): Boolean

    abstract fun delete(i: Int, i2: Int): Boolean

    abstract fun delete(str: String): Boolean

    abstract fun delete(str: String, i: Int, str2: String, i2: Int): Boolean

    abstract fun deleteDb()

    abstract fun getChCount(str: String, i: Int): Int

    abstract fun getGroupDatas(): List<GroupBean>

    abstract fun getGroupDatasByName(str: String): List<GroupBean>

    abstract fun isEmpty(): Boolean

    abstract fun query(str: String): Boolean

    abstract fun query(str: String, i: Int): Boolean

    abstract fun query(str: String, i: Int, str2: String, i2: Int): Boolean

    abstract fun queryByGroupAndDevName(str: String, str2: String, i: Int): Boolean

    abstract fun queryDevName(str: String): Boolean

    abstract fun queryDevNameByDevId(): String

    abstract fun update(str: String, i: Int): Boolean

    abstract fun update(str: String, i: Int, str2: String, i2: Int, i3: Int): Boolean

    abstract fun update(str: String, str2: String): Boolean

    abstract fun updateDevNameById(i: Int, str: String): Boolean
}
